src/DXRMapFixups/DeusEx/Classes/DXRFixupM06.uc

```diff
@@ -1196,8 +1203,21 @@ function AnyEntryMapFixes()
                     break;
             }
         }
+
+        // perhaps the jump in Jock's tone of voice is too much?
+        foreach AllActors(class'#var(prefix)BlackHelicopter', jock) {
+            ces = GetSpeechEvent(GetConversation('M06JockExit').eventList, "I guess we'll find out");
+            ces2 = GetSpeechEvent(GetConversation('JockBarksLumPath').eventList, "Strap in -- we've got a long flight.");
+            ces2.nextEvent = ces.nextEvent;
+            ces.nextEvent = ces2;
+            ces2.speaker = jock;
```

Comment:
This should also set speakerName, in case ConBindEvents gets called again afterwards.  Safer for the future (or any refactoring).